google/cloud/bigquery/v2/minimal/internal/job_configuration_query_test.cc

```diff
@@ -54,7 +54,7 @@ TEST(JobConfigurationQueryTest, DebugString) {
       R"( fields: "clustering-field-2" })"
       R"( destination_encryption_configuration {)"
       R"( kms_key_name: "encryption-key-name" })"
-      R"( script_options { statement_timeout_ms: 10)"
+      R"( script_options { statement_timeout_ms { "10ms" })"
```

Comment:
Having units in the field name when they are also encoded in the value is not recommended.